src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.h

```diff
@@ -101,20 +110,19 @@ typedef struct EnKaizoku {
     /* 0x578 */ Vec3s headRot;
     /* 0x580 */ Vec3f rightFootPos;
     /* 0x58C */ Vec3f leftFootPos;
-    /* 0x598 */ s32 unk_598;
-    /* 0x59C */ s16 unk_59C;
+    /* 0x598 */ s32 cutsceneTimer;
+    /* 0x59C */ s16 cutsceneState; 
     /* 0x59E */ s16 subCamId;
-    /* 0x5A0 */ s16 unk_5A0;
-    /* 0x5A2 */ s16 unk_5A2;
+    /* 0x5A0 */ s16 flashTimer;
     /* 0x5A4 */ Vec3f subCamEye;
     /* 0x5B0 */ Vec3f subCamAt;
     /* 0x5BC */ Vec3f subCamUp;
-    /* 0x5C8 */ Vec3f unk_5C8;
-    /* 0x5D4 */ Vec3f unk_5D4;
-    /* 0x5E0 */ f32 unk_5E0;
-    /* 0x5E4 */ f32 unk_5E4;
-    /* 0x5E8 */ f32 unk_5E8;
-    /* 0x5EC */ f32 unk_5EC;
+    /* 0x5C8 */ Vec3f subCamEyeTarget;
+    /* 0x5D4 */ Vec3f subCamAtTarget;
+    /* 0x5E0 */ f32 subCamVelocity;
+    /* 0x5E4 */ f32 unused5E4;
```

Comment:
Would prefer to just leave this as `unk_5E4` to be perfectly honest. That tends to be how we've been handling unused variables that we can't remove in other actors.

Reply:
I can return it, but I change every variable as part of documentation so I can keep track of what variables I have inspected, leaving a variable completely auto-generated gives a sense that it's completely unfinished